spec/lucky_env_spec.cr

```diff
@@ -19,6 +19,12 @@ describe LuckyEnv do
       ENV["LUCKY_ENV"].should eq "test"
       ENV["DEV_PORT"].should eq "3500"
     end
+
+    it "handles empty values" do
+      results = LuckyEnv.load("./spec/support/.env.test")
```

Comment:
I think we can just move this spec to the `parser_spec.cr`. If this `LUCKY_BLANK=` is in the .env file, then right after [this line](https://github.com/luckyframework/lucky_env/blob/master/spec/parser_spec.cr#L57) you can add this

```crystal
data["LUCKY_BLANK"].should eq ""
```

Let me know if you need more clarification on that.